BacktraceScreen.c

```diff
@@ -0,0 +1,384 @@
+/*
+htop - BacktraceScreen.h
+(C) 2024 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#include "BacktraceScreen.h"
+
+#if defined(HAVE_BACKTRACE)
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/wait.h>
+
+#include "CRT.h"
+#include "FunctionBar.h"
+#include "ListItem.h"
+#include "Macros.h"
+#include "Object.h"
+#include "Panel.h"
+#include "Platform.h"
+#include "Process.h"
+#include "ProvideCurses.h"
+#include "RichString.h"
+#include "Settings.h"
+#include "Vector.h"
+#include "XUtils.h"
+
+
+#define BACKTRACE_FRAME_HEADER_INDEX -1
+
+#define BACKTRACE_PANEL_OPTIONS_NAME_DEMANGLE "Demangle"
+#define BACKTRACE_PANEL_OPTIONS_NAME_RAW "Raw"
+
+#define BACKTRACE_PANEL_OPTIONS_OBJECT_PATH_FULL "Full Path"
+#define BACKTRACE_PANEL_OPTIONS_OBJECT_PATH_BASENAME "Basename"
+
+#define BACTRACE_PANEL_HEADER_NUMBER_FRAME "#"
+#define BACTRACE_PANEL_HEADER_ADDRESS "ADDRESS"
+#define BACTRACE_PANEL_HEADER_NAME "NAME"
+#define BACTRACE_PANEL_HEADER_PATH "PATH"
```

Comment:
Any reason to have these strings defined as macros?

Reply:
Unless used in multiple places, I think they should be removed.

Reply:
I made this decision because it's easier to change the name of column header. Currently only  `BACTRACE_PANEL_HEADER_NUMBER_FRAME` is used twice.
I recognize the name can be shorten.

Reply:
@MrCirdo The header names in htop are usually stored in an array or a table structure, so it's unlikely you need macro tokens for these.

Reply:
Ok interesting, I don't want to use magic value. I will use an array of strings. Thank you for the suggestion.

Reply:
Just take a look at how things are defined in `linux/LinuxProcess.c` for example …